src/App.vue

```diff
@@ -1,23 +1,34 @@
 <script setup lang="ts">
 import { ref } from 'vue';
 
-import IPFSStatus from './components/IPFSStatus.vue'
+import IPFSStatus from './components/IPFSStatus.vue';
+import OrcestraLogoLight from './images/orcestra-logo.svg';
+import OrcestraLogoDark from './images/orcestra-logo-dark.svg';
+
+const OrcestraLogo = window.matchMedia('(prefers-color-scheme: dark)').matches ? OrcestraLogoDark : OrcestraLogoLight;
```

Comment:
I'd suggest to skip the JS-based logo selection in favor of a pure CSS solution. In particular, because this matchMedia query will not be re-evaluated in case the color scheme changes over time. And there are at least some people which change their color scheme based on day/night cycle.

```suggestion
import OrcestraLogo from './images/orcestra-logo.svg';
```

Reply:
That's a really neat solution! 👌